src/providers/MeetingProvider/types.ts

```diff
@@ -17,6 +17,7 @@ export interface MeetingJoinData {
 
 export interface AttendeeResponse {
   name?: string;
+  [attribute: string]: any;
```

Comment:
Could you please check if this also affects the [`RosterAttendeeType`](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/types/index.ts#L48). Because, the `RosterProvider` provides `RosterType` state. This `RosterType` holds mapping from a string `attendeeId` to `RosterAttendeeType`. The extra keys now the `AttendeeResponse` provides may not be set (may be errors out) since the `RosterAttendeeType` does not have those keys. Just making sure if you have tested this?

Reply:
Yes, with only the change of `AttendeeResponse`, the extra data can be received in `RosterProvider` and be displayed using `RosterCell`, no error is out.